testing/test_conftest.py

```diff
@@ -208,11 +211,11 @@ def test_conftest_symlink(testdir):
     # /symlinktests -> /real/app/tests (running at symlinktests should fail)
     # /symlink -> /real (running at /symlink should work)
 
-    real = testdir.tmpdir.mkdir("real")
+    real = pytester.mkdir("real")
     realtests = real.mkdir("app").mkdir("tests")
```

Comment:
> testing/test_conftest.py:213: error: Argument 1 to "mkdir" of "Path" has incompatible type "str"; expected "int" [arg-type]

```real``` is a instance of ```Path```. ```Path``` dosen't support making directories like this ```real.mkdir("app").mkdir("tests")```. You can first create a path and then make it into directories using method ```.mkdir(parents=True)```.
```suggestion
    realtests = real.joinpath("app/tests")
    realtests.mkdir(parents=True)
```